### DISCUSSION TITLE
longhorn backup issue mounting nfs

### DISCUSSION THREAD

**Discussion by jogaleins** [longhorn/longhorn#3805] [Q&A]
Hi, 

Anyone experience this issue. I tried manually mounting the nfs on my nodes and it's working. It was working before then suddenly I have this issue. Thanks!

LONGHORN VERSION: 1.2.0

K3S:
NAME     STATUS   ROLES                  AGE    VERSION        INTERNAL-IP     EXTERNAL-IP   OS-IMAGE             KERNEL-VERSION      CONTAINER-RUNTIME
node4    Ready    <none>                 50d    v1.20.4+k3s1   192.168.2.209   <none>        Alpine Linux v3.14   5.10.93-0-virt      containerd://1.4.3-k3s3
node3    Ready    <none>                 196d   v1.20.4+k3s1   192.168.2.208   <none>        Alpine Linux v3.14   5.10.93-0-virt      containerd://1.4.3-k3s3
node1    Ready    control-plane,master   196d   v1.20.4+k3s1   192.168.2.202   <none>        Alpine Linux v3.14   5.10.93-0-virt      containerd://1.4.3-k3s3
unode1   Ready    <none>                 23d    v1.20.4+k3s1   192.168.2.204   <none>        Ubuntu 20.04.4 LTS   5.4.0-105-generic   containerd://1.4.3-k3s3
node2    Ready    <none>                 150d   v1.20.4+k3s1   192.168.2.203   <none>        Alpine Linux v3.14   5.10.109-0-virt     containerd://1.4.3-k3s3

ERROR LOG:
error listing backup volume names: Failed to execute: /var/lib/longhorn/engine-binaries/longhornio-longhorn-engine-v1.2.0/longhorn [backup ls --volume-only nfs://192.168.2.99:/kubedata], output Cannot mount nfs 192.168.2.99:/kubedata: Failed to execute: mount [-t nfs4 -o nfsvers=4.2 192.168.2.99:/kubedata /var/lib/longhorn-backupstore-mounts/192_168_2_99/kubedata], output mount.nfs4: mounting 192.168.2.99:/kubedata failed, reason given by server: No such file or directory , error exit status 32 , stderr, time="2022-03-31T06:06:52Z" level=error msg="Cannot mount nfs 192.168.2.99:/kubedata: Failed to execute: mount [-t nfs4 -o nfsvers=4.2 192.168.2.99:/kubedata /var/lib/longhorn-backupstore-mounts/192_168_2_99/kubedata], output mount.nfs4: mounting 192.168.2.99:/kubedata failed, reason given by server: No such file or directory\n, error exit status 32" , error exit status 1

**Comment by derekbit** [comment]
Do you add the `fsid` in your nfs server config?

**Reply by jogaleins** [reply]
@derekbit - this is the config of my exports file
/export/kubedata 192.168.2.0/24(rw,sync,no_subtree_check)

**Reply by derekbit** [reply]
Got it. Your backup target path should be `nfs://192.168.2.99:/export/kubedata`

**Comment by jogaleins** [comment]
hi Derek, Thanks, you're right my need to add /export at the beginning.

On Thu, Mar 31, 2022 at 3:21 PM Derek Su ***@***.***> wrote:

> Do you add the fsid in your nfs server config?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/longhorn/longhorn/discussions/3805#discussioncomment-2475995>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APA4PMXT6NXK5NR6KYCVLQDVCVG57ANCNFSM5SEIIPNQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Regards,
Adrian
